rcl/test/rcl/test_timer.cpp

```diff
@@ -50,13 +50,19 @@ class TestTimerFixture : public ::testing::Test
 
 TEST_F(TestTimerFixture, test_two_timers) {
   rcl_ret_t ret;
+
+  rcl_clock_t clock;
+  rcl_allocator_t allocator = rcl_get_default_allocator();
+  ret = rcl_clock_init(RCL_STEADY_TIME, &clock, &allocator);
```

Comment:
Recommend finishing tests with `rcl_*_clock_fini()` to reduce false-positive address sanitizer output.

Reply:
Absolutely - done in 0c972079e41d9a8730b1da4643aa0903dfaf2857.